src/components/PayPalButtons.js

```diff
@@ -45,7 +45,12 @@ export default function PayPalButtons(props) {
         });
 
         return cleanup;
-    });
+    }, [
+        isResolved,
+        props.forceReRender,
+        props.fundingSource,
+        props.shippingPreference,
```

Comment:
Thanks for updating this. Sorry to bug you with another change but could you also add the style prop like so?

```suggestion
        props.shippingPreference,
        props.style,
```